index.html

```diff
@@ -48,13 +63,149 @@ <h3>LET'S CONNECT</h3>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
+            </div>
+           </div>
+
+           <div class="mobile-version">
+            <div class="mobile-section-div-left">
+              <h1>Uber Navigation</h1>
+              <div class="canopy-div">
+                <p>Uber</p>
+                <img src="Counter.svg" alt="" />
+                <p>Lead developer</p>
+                <img src="Counter.svg" alt="" />
+                <p>2018</p>
+              </div>
+             <p class="writeup-div">A daily selection of privately personalized reads; no accounts or sign ups required</p>
+             <button class="desktop-btn">HTML</button>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
+            </div>
+            <div class="mobile-image-right">
+              <img src="desktop2.svg" alt="" class="card-img-img">
+            </div>
+           </div>
+        </section>
+
+        <section class="desktop-second-about">
+          <div class="desktop-about-left">
+            <h1 class="abouth1">About Myself</h1>
+            <p>Hello im a software developer, i can help you build a product, feature or website look through some of my work
+              and experience! if you like what you see and have a project you need coded, dont hesitate to contact me.
+            </p>
+            <h2>LET'S CONNECT</h2>
+                <div class="desktop-icon-div">
+                    <ul>
+                        <li><a href=""><img src="Vector (2).svg" alt="" /></a></li>
+                        <li><a href=""><img src="Linkedin icon.svg" alt="" /></li>
+                        <li><a href=""><img src="Vector (3).svg" alt="" /></a></li>
+                        <li><a href=""><img src="Vector (4).svg" alt="" /></a></li>
+                        <li><a href=""><img src="Vector (5).svg" alt="" /></a></li>
+                    </ul>
+                </div>
+                <button class="resume-btn">Get my resume</button>
+          </div>
+          <div class="desktop-about-right">
+            <h2>Languages</h2>
+            <div class="stack-div">
+              <div class="stack-java">
+                <img src="Ellipse 1.svg" alt="" />
+                 <p>JavaScript</p>
+              </div>
+              <div class="stack-java">
+                <img src="Ellipse 1 (3).svg" alt="" />
+                <p>HTML</p>
+              </div>
+              <div class="stack-java">
+                <img src="Ellipse 1 (2).svg" alt="">
+                <p>CSS</p>
+              </div>
+            </div>
+            <p>Frameworks</p>
+              <hr>
+              <h2>Skills</h2>
+              <hr>
+          </div>
+        </section>
+
+        
```

Comment:
- [ ] I'm putting this here because your contact section is not available to write comments on.
- [ ] As seen in the Figma template. Kindly make sure that `get in touch` button is centered on the page.

|  |  |
| ------------- | ------------- |
| Your Design | <img width="598" alt="image" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/6fdb25e4-50cb-4aca-9347-1b197c8b2b25"> |
| Figma Design  | <img width="559" alt="image" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/6bdfcfb5-27cd-426b-bce0-125089ea9107">  |